drivers/modem/ublox-sara-r4.c

```diff
@@ -82,7 +91,11 @@ static const struct mdm_control_pinconfig pinconfig[] = {
 #define MDM_CMD_SEND_TIMEOUT		K_SECONDS(10)
 #define MDM_CMD_CONN_TIMEOUT		K_SECONDS(31)
 #define MDM_REGISTRATION_TIMEOUT	K_SECONDS(180)
+#if defined(CONFIG_MODEM_UBLOX_SARA_U2)
+#define MDM_PROMPT_CMD_DELAY		K_MSEC(50)
```

Comment:
I have found 75 MSEC has worked well as the spec states you should wait at least 50ms after the @ prompt.